rocketpy/simulation/flight_data_importer.py

```diff
@@ -0,0 +1,398 @@
+            The path to the file we are reading from.
+
+        Returns
+        -------
+        None
+
+        Raises
+        ------
+        ValueError
+            If the time column is not found in the header of the file.
+        """
+        if self._columns_map[filepath]:
+            for col, atr in self._columns_map[filepath].items():
+                if atr.lower() == "time":
+                    self._time_cols[filepath] = col
+                    return None
+        if "time" in self._columns[filepath]:
+            self._time_cols[filepath] = "time"
+            return None
+        raise ValueError(
+            "Unable to determine the time column, please specify it. The time column "
+            + "must be in seconds and must be named 'time'. Alternatively, you "
+            + "can specify the time column using the 'columns_map' argument. "
+            + "Available columns are: "
+            + str(self._columns[filepath])
+        )
+
+    def __create_attributes(self, filepath, interpolation, extrapolation):
+        """Creates the attributes to emulate a Flight object.
+
+        Parameters
+        ----------
+        filepath : str
+            The path to the file we are reading from.
+        interpolation : str
+            The interpolation method to use when setting the rocketpy.Function
+            objects.
+        extrapolation : str
+            The extrapolation method to use when setting the rocketpy.Function
+            objects.
+
+        Raises
+        ------
+        ValueError
+            If the time column is not found in the header of the file.
+        """
+        time_col = self._time_cols[filepath]
+        units = self._units[filepath]
+
+        # Extract time values
+        try:
+            times = self._data[filepath][
+                :, self._original_columns[filepath].index(time_col)
+            ]
+        except ValueError as e:
+            raise ValueError(
+                f"Unable to find column '{time_col}' in the header of the file."
+                + "The available columns are:"
+                + str(self._original_columns[filepath])
+            ) from e
+
+        created = []
+        for col, name in self._columns_map[filepath].items():
+            if name == "time":  # Handle time separately
+                setattr(self, name, times)
+                created.append(name)
+                continue
+
+            # Find the index of the current column in the filtered data
+            try:
+                col_idx = self._original_columns[filepath].index(col)
+            except ValueError:
+                warnings.warn(
+                    f"Unable to find column '{col}' in the header of the file. "
+                    + f"The attribute '{name}' won't be set. The available "
+                    + "columns are:"
+                    + str(self._original_columns[filepath]),
+                    UserWarning,
+                )
+                continue
```

Comment:
This warning becomes annoying when working with directories instead of files.
Additionally, it is a bit irrelevant I think. 

I'm going to remove it.

Reply:
Solved here: [cf50451](https://github.com/RocketPy-Team/RocketPy/pull/550/commits/cf50451fae5ddb092b0ff35153e39eeea5d566d5)